devito/operator.py

```diff
@@ -124,31 +123,32 @@ def __init__(self, expressions, **kwargs):
         # Finish instantiation
         super(Operator, self).__init__(self.name, nodes, 'int', parameters, ())
 
-    @cached_property
-    def _argument_defaults(self):
+    def _argument_defaults(self, arguments):
```

Comment:
So this method is useless at this point. I think you can take the code and put it directly into ``def arguments``.